File: app/src/main/java/com/meteoro/testeandroid/core/data/model/Cell.java
package com.meteoro.testeandroid.core.data.model;

import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.experimental.Accessors;

@Accessors(fluent = true, chain = true)
public class Cell {

    @Getter
    int id;

    @Getter
    int type;

    @Getter
    String message;

    @Getter
    @SerializedName("typefield")
    String typeField;

    @Getter
    boolean hidden;

    @Getter
    int topSpacing;

    @Getter
    int show;

    @Getter
    boolean required;
}

File: app/src/main/java/com/rafhack/testeandroid/data/entities/Cell.kt
package com.rafhack.testeandroid.data.entities

data class Cell(
        var id: Int = 0,
        var type: Int = 0,
        var message: String = "",
        var typefield: String? = "",
        var hidden: Boolean = false,
        var topSpacing: Int = 0,
        var show: Int? = 0,
        var required: Boolean = false
)